test/files/run/macro-term-declared-in-anonymous/Macros_Test_2.scala

```diff
@@ -1,3 +1,5 @@
+// scalac: -language:experimental.macros
+//import scala.language.experimental.macros
```

Comment:
NB the import results in circular dependency error, so the option flag must be used.